src/styles/index.scss

```diff
@@ -1,3 +1,254 @@
+$header-height: 60px;
+$hover-color: #00acdc;
+$nav-link-color: #060b35;
+
+$buy-button-color: rgb(0, 172, 220);
+$product-att-color: rgb(97, 96, 112);
+
+:root {
+  --column-count: 1;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+// #region header
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+  .logo {
+    height: 40px;
+    width: 40px;
+    object-fit: contain;
+    padding: 10px 0;
+  }
+
+  nav {
+    text-transform: uppercase;
+
+    .nav__list {
+      display: flex;
+      margin: 0;
+      padding: 0;
+      list-style: none;
+
+      .nav_item {
```

Comment:
The class name '.nav_item' should be written in kebab-case as '.nav-item' according to the BEM naming convention.